<launch>
<!--
    <node
        pkg="nomad"
        type="imu_publisher.py"
        name="imu_publisher"
        respawn="true"
        >

        <param name="imu_port"        value="/dev/ttyUSB0"/>
        <param name="imu_baud_rate"   value="38400"/>
        <param name="imuUpdateHz"     value="10.0"/>
    </node>

 -->

    <node
        pkg="robot_pose_ekf"
        type="robot_pose_ekf"
        name="robot_pose_ekf"
        respawn="true"
        >

        <!--
          Since robot_pose_ekf subscribes to "odom" by
          default, it must be adjusted to instead subscribe
          to "wheel_odom", so that robot_pose_ekf can be the
          node which publishes to "odom".
          -->
        <remap from="odom"           to="wheel_odom" />

        <param name="output_frame"   value="odom"/>
        <param name="freq"           value="1.0"/>
        <param name="sensor_timeout" value="1.0"/>  
        <param name="odom_used"      value="true"/>
        <param name="imu_used"       value="true"/>
        <param name="vo_used"        value="false"/>
        <param name="debug"          value="true"/>
        <param name="self_diagnose"  value="false"/>
    </node>

</launch>
